src/TLuaInterpreter.cpp

```diff
@@ -1365,20 +1365,25 @@ int TLuaInterpreter::paste( lua_State * L )
 
 int TLuaInterpreter::feedTriggers( lua_State * L )
 {
-    string luaWindowName="";
-    if( lua_isstring( L, 1 ) )
-    {
-        luaWindowName = lua_tostring( L, 1 );
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if (! pHost) {
+        lua_pushstring(L, tr("feedTriggers: NULL Host pointer - something is wrong!")
```

Comment:
These `tr(...)` strings are always unique because the first part is the function name, so I guess that substituting `QStringLiteral(...)` will be a drop in replacement (except for those few cases with a second argument) - would it be an acceptable compromise to switch to those; without the hassle of splitting up each literal chunk into a `QLatin1String`?